pyproject.toml

```
@@ -1,7 +1,85 @@
 [build-system]
 requires = [
-    "cython>=0.29.29",
+    "Cython~=0.29",
```

We need a fix introduced in 0.29.29.

```suggestion
    "Cython~=0.29.29",
```